views/billing_report_view.sql

```diff
@@ -11,8 +11,8 @@ SELECT
     IF(iseq_run_lane_metrics.instrument_model = 'NovaSeq', ExtractValue(iseq_run_info.run_parameters_xml, '//SbsCycleKit'), NULL)
                                                AS sbs_cycle_kit,
     IF(INSTR(ExtractValue(iseq_run_info.run_parameters_xml, '//RecipeVersion'), '_CustomPrimer_') > 0, 'Yes', 'No') AS custom_primer_used,
-
-
+    SUBSTRING_INDEX(ExtractValue(iseq_run_info.run_parameters_xml, '//ConsumableInfo[Type="Reagent"]/Name'), ' ', 1) AS kit_type,
```

Comment:
Are we referring to [`EXTRACTVALUE`](https://mariadb.com/kb/en/extractvalue/) String function here? It is worth making it upper case?

Reply:
I will update that for better readability